doc/classes/VisualServer.xml

```diff
@@ -1205,6 +1236,7 @@
 			<argument index="1" name="scale" type="float">
 			</argument>
 			<description>
+				Sets a custom fov for the background [Sky]. Equivalent to [member Environment.background_sky_custom_fov].
```

Comment:
Capitalize FOV, and maybe define it (unless it's done in the Environment method already).